### PR TITLE
Devices api

### PR DESCRIPTION
**Pull request by bhaveshpatel88**

### CODE REVIEW

**Comment by carsten989** [comment]
Should this change to the REST API make it into the owgw schema in openapi/owgw.yaml?

**Review by carsten989 — APPROVED** [review]
Looks good, other than the missing openapi schema change

**Comment by acapparelli** [comment]
I thought we were changing the parameter to includeProvisioned=true/false rather than an empty param?

**Review by acapparelli — APPROVED** [review]

**Comment by bhaveshpatel88** [comment]
Missed my last commit, added the schema and changed to includeProvisioned=false/true

File: src/storage/storage_device.cpp [hunk 1]
```diff
@@ -195,17 +195,32 @@ namespace OpenWifi {
...
+										 const std::string &platform, bool includeProvisioned) {
 		try {
 			Poco::Data::Session Sess = Pool_->get();
 			Poco::Data::Statement Select(Sess);
 
 			std::string st;
+			std::string whereClause = "";
 			if(!platform.empty()) {
-				st = "SELECT SerialNumber From Devices WHERE DeviceType='" + platform + "' ";
+				if (includeProvisioned == false) {
```
Comment (carsten989) [inline src/storage/storage_device.cpp:206]: not to nitpick but this probably should be if (!includeProvisioned) {

**Pull request merged**